### PR TITLE
Update param filenames

### PR DESCRIPTION
**Pull request by AlexAxthelm**
Update filenames for default parameter files, to avoid name collisions while inheriting (`analysis_2023Q4.json` and `reporting_2023Q4.json`, rather than both being named `2023Q4.json`)

Depends on:
- [x] https://github.com/RMI-PACTA/pacta.workflow.utils/pull/37

Relates to:
- [x] https://github.com/RMI-PACTA/workflow.pacta/pull/100

### CODE REVIEW

File: inst/extdata/schema/rawParameters.json [hunk 1]
```diff
@@ -11,13 +11,7 @@
     },
     "inherit": {
       "type": "string",
-      "description": "Path to a file with inheritance information.",
-      "enum": [
-        "GENERAL",
-        "GENERAL_2022Q4",
-        "GENERAL_2023Q4",
-        "default"
-      ]
```
Comment (jdhoffa) [inline inst/extdata/schema/rawParameters.json:20]: Why are you removing this `enum` key here?

File: inst/extdata/scripts/run_pacta_report.R [hunk 2]
```diff
@@ -14,7 +14,8 @@ params <- pacta.workflow.utils::parse_raw_params(
   raw_schema_file = system.file(
     "extdata", "schema", "rawParameters.json",
     package = "workflow.pacta.report"
-  )
+  ),
+  force_array = c("portfolio", "files")
```
Comment (jdhoffa) [inline inst/extdata/scripts/run_pacta_report.R:18]: Can you explain to me what this does/ why it is necessary?

**Review by jdhoffa — APPROVED** [review]
A few questions (mainly for my own understanding) but otherwise looks good

File: inst/extdata/schema/rawParameters.json (hunk 1 above)
Comment (AlexAxthelm) [reply inst/extdata/schema/rawParameters.json:20]: I don't want to worry about sorting out the multi-inheritence with conflicting `enum`s (example, If we're inheriting `c("analysis_2023Q4", "reporting_2023Q4")` then there's going to be a conflict with the enums in this repo and `workflow.pacta`)

File: inst/extdata/scripts/run_pacta_report.R (hunk 2 above)
Comment (AlexAxthelm) [reply inst/extdata/scripts/run_pacta_report.R:18]: I removed the default in https://github.com/RMI-PACTA/pacta.workflow.utils/pull/38, so this is adding that behavior back in.

**Pull request merged**